phase1/vsphere/Kconfig

```diff
@@ -0,0 +1,73 @@
+menu "vSphere configuration"
+
+config phase1.vSphere.url
+	string "vCenter url"
+	help
+		The vCenter Url
+
+config phase1.vSphere.port
+	string "vCenter port"
+	default 443
+	help
+		The vCenter port
+
+config phase1.vSphere.username
+	string "vCenter username"
+	help
+		The vCenter Username
+
+config phase1.vSphere.password
+	string "vCenter password"
+	help
+		The vCenter password
+
+config phase1.vSphere.insecure
+	string "Does host use self-signed cert"
+	default true
+	help
+		If the host uses a self-signed cert.
+
+config phase1.vSphere.resourcePool
```

Comment:
We don't know if plugin works with multiple resources pool. Moreover, there is no way to mention resource pool in plugin.